<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>1004</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить преподавателя:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="surnameEdit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>70</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во бюджетников:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во платников:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="budgetSpin">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="paidSpin">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="studentEdit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить студента:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="paidRadio">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>350</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Платник</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">studentGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="budgetRadio">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>350</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Бюджетник</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">studentGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="addTeacherButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить препода</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addStudentButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить студента</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>1281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="interestTable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Компьютерная графика</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Моделирование</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LMS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Базы данных</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Операционные системы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Компьютерные сети</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Криптография</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Машинное обучение</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Анализ данных</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Блокчейн</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Чилл</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>510</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Научные интересы:</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>1291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>510</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать любимого препода:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>510</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать любимого студента:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="teachTable">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>560</y>
      <width>351</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>500</y>
      <width>20</width>
      <height>371</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>500</y>
      <width>20</width>
      <height>371</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="studentTable">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>560</y>
      <width>351</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matchButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>910</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Match!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1289</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="studentGroup"/>
 </buttongroups>
</ui>
